Gemfile

```diff
@@ -61,6 +61,7 @@ gem 'easypost', '>=3.0.1'
 gem 'StreetAddress', github: 'Exygy/street-address', require: 'street_address'
 
 # google translation
+gem 'faraday'
```

Comment:
suggestion: we don't actually need this anymore, but it shouldn't make a difference